lib/availability_status_listener.rb

```diff
@@ -39,6 +40,11 @@ def process_event(event)
     Rails.logger.info("Kafka message #{event.message} received with payload: #{event.payload}")
     return unless event.message == EVENT_AVAILABILITY_STATUS
 
+    if (REQUIRED_HEADERS - event.headers.keys).any?
+      Rails.logger.error("Kafka message missing required header(s): #{(REQUIRED_HEADERS - event.headers.keys).join(',')}; returning.")
```

Comment:
I would add some identification, i.e. `event.message` (and maybe the payload too?)

Reply:
That gets logged on line 40 right away at the beginning of processing. Is that ok?

Reply:
When the event.message will be there too, then you can filter messages by the string in the Kibana so it will be more comfortable for troubleshooting

Reply:
Ahh yeah good point, added!